zebra-consensus/src/transaction.rs

```diff
@@ -518,31 +499,66 @@ where
                 //.map_err(|e| BoxError::from(Box::new(e)))?;
             }
 
-            let _rsp = redjubjub_verifier
-                .ready_and()
-                .await?
-                .call((bvk, sapling_shielded_data.binding_sig, &shielded_sighash).into())
-                .boxed();
-
             // TODO: stop ignoring binding signature errors - #1939
-            // async_checks.push(rsp);
+            // async_checks.include(
+            //     primitives::redjubjub::VERIFIER
+            //         .clone()
+            //         .oneshot((bvk, sapling_shielded_data.binding_sig, &shielded_sighash).into()),
+            // );
         }
 
         Ok(async_checks)
     }
+}
+
+/// A set of asynchronous checks that should succeed.
```

Comment:
Missing documentation: Let's explicitly say that the checks are unordered.

Reply:
Done in [c340478](https://github.com/jvff/zebra/commit/c340478bef309061e1e9752d464ffe7aba8aabe9).

Reply:
Nit: `check` is the place where ordering matters, but it's not documented there.

This is not a blocker to merging.